ext/src/http/client/curl/http_client_curl.cc

```diff
@@ -258,7 +258,7 @@ void HttpClient::CleanupSession(uint64_t session_id)
       else if (session->IsSessionActive() && session->GetOperation())
       {
         // If this session is alread waiting to be removed, just wakeup background thread to call
-        // doRemoveSessions()
+        pending_to_abort_sessions_[session_id] = std::move(session);
```

Comment:
The comment had 2 lines, one of them removed, so now it reads:

// If this session is alread waiting to be removed, just wakeup background thread to call

Please reword it, and fix typo (alread -> already) while at it.

If I understood correctly:

// If this session is already waiting to be removed, give it to the background thread for cleanup.

Reply:
Thanks.